lib/raidenclient/RaidenClient.ts

```diff
@@ -75,19 +78,23 @@ class RaidenClient extends BaseClient {
   public readonly type = SwapClient.Raiden;
   public readonly cltvDelta: number = 0;
   public address?: string;
+  public currencies: { [key: string]: string } = {};
```

Comment:
This can be a `Map` type as well, and maybe it would make more sense to call this `tokenAddresses` since that's what the values are?